DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: stoner
 Title: Support for building VIMC Montagu Touchstones, using dettl
-Version: 0.0.2
+Version: 0.0.3
 Authors@R:
-    c(person("Wes", "Hinsley",role = c("aut", "cre", "itr"),
+    c(person("Wes", "Hinsley",role = c("aut", "cre", "dnc", "itr"),
```

Comment:
:dancer:

Reply:
This can only get more exciting the more stoner PRs we get through.